History.md

```diff
@@ -4,6 +4,7 @@ unreleased
   * Fix incorrect middleware execution with unanchored `RegExp`s
   * Fix `res.jsonp(obj, status)` deprecation message
   * Fix typo in `res.is` JSDoc
+  * deps: proxy-addr@2.0.6
```

Comment:
Users will observe a change in the `ipaddr.js` dependency, so be sure to list that has a sub change here (see previous change logs for example).

Reply:
`proxy-addr@2.0.6` use `ipaddr.js@1.9.1` and i see that  `ipaddr.js@1.9.1` haven't any api changes. you can check [this](https://github.com/whitequark/ipaddr.js/releases/tag/v1.9.1).

Reply:
Right. The observable change in this case is just listing the ipaddr.js version is going to change in the dependency tree. Check out the previous change logs here for examples of listing that.